docs/pages/auto-discovery/introduction.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Teleport Auto-Discovery
+description: "Learn how to use the Teleport Discovery Service, which automatically enrolls resources by query APIs"
+---
+
+The Teleport Discovery Service automatically detects resources in your
+infrastructure and enrolls them in your Teleport cluster. When you deploy
+servers, databases, and Kubernetes clusters, Teleport enables secure access to
+these resources with no further configuration. This lets you decouple the need
+to protect your infrastructure resources from the work of deploying and managing
+them.
+
+The Discovery Service runs on [Teleport agents](../agents/introduction.mdx). It
+periodically queries cloud provider APIs to list resources in your
+infrastructure. It then reconciles these resources with Teleport resources
+registered on the Auth Service backend.
+
+Set up Teleport auto-discovery for resources in your infrastructure:
+
+- [Servers](./servers.mdx)
+- [Kubernetes clusters](./kubernetes.mdx)
+- [Databases](./databases.mdx)
```

Comment:
I think the Kubernetes application discovery guide should also be moved there: https://goteleport.com/docs/application-access/enroll-kubernetes-applications/get-started/